migrations/20181211174223-create-user.js

```diff
@@ -14,7 +19,9 @@ module.exports = {
         type: Sequelize.STRING,
       },
       gender: {
-        type: Sequelize.STRING,
+        type: Sequelize.ENUM({
```

Comment:
Gender is defined as `ENUM` in the models but it was erroneously defined as`STRING` in the migration.
I fixed this in here.